### PR TITLE
Add Url property to Page struct

### PR DESCRIPTION
**Pull request by adwinying**
Hey there, 
Notion has added a `url` property to the [Page object](https://developers.notion.com/reference/page) and I tried to add it to the Page struct.

I'm very new to Go so please tell me if there's any mistakes.

### CODE REVIEW

File: page.go [hunk 1]
```diff
@@ -66,6 +66,7 @@ type Page struct {
 	Archived       bool       `json:"archived"`
 	Properties     Properties `json:"properties"`
 	Parent         Parent     `json:"parent"`
+	Url            Url        `json:"url"`
```
Comment (jomei) [inline page.go:69]: Let's call it  `URL` instead of `Url`

File: page.go [hunk 2]
```diff
@@ -86,6 +87,8 @@ type PageCreateRequest struct {
 	Children   []Block    `json:"children,omitempty"`
 }
 
+type Url string
```
Comment (jomei) [inline page.go:90]: It seems like we don't need the type alias here. Let's parse the `url` as a string

**Review by jomei — CHANGES_REQUESTED** [review]
Oh, I've missed that. Thanks a lot for this PR! 🔥

File: page.go (hunk 1 above)
Comment (adwinying) [reply page.go]: Changed `Url` to `URL`

File: page.go (hunk 2 above)
Comment (adwinying) [reply page.go]: Changed `URL` property type to `string`

**Review by jomei — APPROVED** [review]
Awesome, Thanks!

**Pull request merged**